yarn-project/pxe/src/contract_function_simulator/oracle/private_execution_oracle.ts

```diff
@@ -364,6 +364,7 @@ export class PrivateExecutionOracle extends UtilityExecutionOracle {
    * @param argsHash - The arguments hash to pass to the function.
    * @param sideEffectCounter - The side effect counter at the start of the call.
    * @param isStaticCall - Whether the call is a static call.
+   * @param skipClassVerification - Whether to skip class verification.
```

Comment:
```suggestion
```

Seems to have been placed here by mistake instead of in the `runUtility` function.